src/svcutilcore/dir.props

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <AssemblyKey>Open</AssemblyKey>
+    <IsNETCoreApp>true</IsNETCoreApp>
+    <IsNETCoreAppRef>true</IsNETCoreAppRef>
```

Comment:
I am wondering if `IsNETCoreAppRef` should be false as svcutil is a tool and does not expose any APIs, but I may not understand `IsNETCoreAppRef` correctly.